includes/admin/promos/notices/class-license-upgrade-notice.php

```diff
@@ -148,8 +170,10 @@ protected function _display() {
 				// No license keys active at all.
 				printf(
 				/* Translators: %1$s opening anchor tag; %2$s closing anchor tag */
-					__( 'You are using the free version of Easy Digital Downloads. %1$sPurchase a pass%2$s to get email marketing tools and recurring payments.', 'easy-digital-downloads' ),
+					__( 'You are using the free version of Easy Digital Downloads. %1$sPurchase a pass%2$s to get email marketing tools and recurring payments. %3$sAlready have a Pass?%4$s', 'easy-digital-downloads' ),
 					'<a href="' . esc_url( add_query_arg( $this->query_args( 'core', $source ), 'https://easydigitaldownloads.com/pricing/' ) ) . '" target="_blank">',
+					'</a>',
+					'<a href="' . esc_url( add_query_arg( $this->query_args( 'core', $source ), 'https://easydigitaldownloads.com/what-is-an-edd-pass' ) ) . '" target="_blank">',
```

Comment:
Should this link be added to the next condition as well, if someone has a licensed product but not a pass?

Reply:
No because once their key is a pass, the entire system should identify that at the next check.

Reply:
When I upgraded to a pass, it had an entirely new license key, so I _think_ if you have a single product key entered and buy a pass but don't enter the key, you'll still see the single product message.